samples/CMakeLists.txt

```diff
@@ -22,6 +23,9 @@ foreach(src_file ${SAMPLES_LIST})
   else()
     set_target_properties(${sample_exec} PROPERTIES CXX_STANDARD 14)
   endif()
+  target_compile_definitions(${sample_exec} PRIVATE -D${TUNING_TARGET})
+  target_compile_definitions(${sample_exec} PUBLIC GEMM_VECTORIZATION_SUPPORT=1)
+
```

Comment:
Needed to select the appropriate backend and to enable vectorization on the sample app.